rotkehlchen/exchanges/iconomi.py

```diff
@@ -224,15 +229,11 @@ def query_balances(self, **kwargs: Any) -> ExchangeQueryBalances:
             try:
                 asset = asset_from_iconomi(ticker)
 
-                # There seems to be a bug in the ICONOMI API regarding balance_info['value'].
-                # The value is supposed to be in USD, but is actually returned
-                # in EUR. So let's use the Inquirer for now.
                 try:
-                    usd_price = Inquirer().find_usd_price(asset=asset)
-                except RemoteError as e:
-                    self.msg_aggregator.add_error(
-                        f'Error processing ICONOMI balance entry due to inability to '
-                        f'query USD price: {str(e)}. Skipping balance entry',
+                    usd_value = deserialize_fval(balance_info['value'], 'usd_value', 'iconomi')
+                except DeserializationError as e:
```

Comment:
you should also catch `KeyError` here since `balance_info['value']` may raise if `value` is missing.